src/FEValues/common_values.jl

```diff
@@ -117,6 +136,7 @@ curl_from_gradient(∇v::SecondOrderTensor{3}) = Vec{3}((∇v[3,2] - ∇v[2,3],
 
 """
     function_value(fe_v::AbstractValues, q_point::Int, u::AbstractVector)
+    function_value(iv::InterfaceValues, q_point::Int, u::AbstractVector, dof_range = eachindex(u); here::Bool = true)
```

Comment:
Why do we need `dof_range` here? Also, note the possible name collision with the function.

Docstring for `here`